db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_02_13_210834) do
+ActiveRecord::Schema.define(version: 2019_04_18_180154) do
```

Comment:
This could cause issues on prod and other existing dev environments, as Rails uses the timestamp to determine if there are migrations that need running. To be safe, let's rename `db/migrate/20190418180154_baseline.rb` to `db/migrate/20190213210834_baseline.rb`. This uses the timestamp of the last run migration before you baselined it, so to other CEQR App instances, it'll just assume its db is at baseline (which it is.)

Reply:
Once the migration file is renamed, rerun `rails db:migrate` to make sure the schema file shows the correct, older timestamp. At that point, `schema.rb` shouldn't actually have a diff.

Reply:
👍